main/src/com/google/refine/exporters/sql/SqlCreateBuilder.java

```diff
@@ -82,13 +83,21 @@ public String getCreateSQL() {
       
                 
                 if (name != null) {
-                    if(trimColNames) {
+                	if(trimColNames) {
```

Comment:
There seem to be whitespace changes mixed in with functional changes. Please don't reformat existing code and use 4 space indents for new code (to match the existing code in the module).